src/components/mobile-menu-container/mobile-menu-container.module.css

```diff
@@ -6,17 +6,29 @@
 ***
 */
 .root {
-  background-color: white;
-  box-shadow: 0 2px 3px rgba(101, 106, 118, 0.1),
-    0 12px 28px rgba(101, 106, 118, 0.25);
-  display: none;
-  flex-direction: column;
-  flex-shrink: 0;
-  height: calc(100vh - var(--sticky-bars-height));
-  left: -150vw;
-  overflow-y: auto;
-  position: fixed;
-  top: var(--sticky-bars-height);
-  width: 100%;
-  z-index: 1;
+	background-color: white;
+	box-shadow: 0 2px 3px rgba(101, 106, 118, 0.1),
+		0 12px 28px rgba(101, 106, 118, 0.25);
+	display: none;
+	flex-direction: column;
+	flex-shrink: 0;
+	height: calc(100vh - var(--sticky-bars-height));
+	left: -150vw;
+	overflow-y: auto;
+	position: fixed;
+	top: var(--sticky-bars-height);
+	width: 100%;
+	z-index: 1;
+}
+
+.mobileAuthenticationControls {
```

Comment:
It seems like these buttons are rendering at desktop size also? 
<img width="1758" alt="Screen Shot 2022-07-27 at 9 43 49 AM" src="https://user-images.githubusercontent.com/36613477/181304191-bf3b0819-dabb-4f28-8d2e-597df733c758.png">

Reply:
Gah! Great catch. I forgot to add the `g-hide-on-desktop` helper class. 😆

Reply:
🗒️ Fixed in https://github.com/hashicorp/dev-portal/pull/742/commits/5fb1b30cae60c3916881d837b3a57681d70722e5